syncmymoodle/__main__.py

```diff
@@ -1172,6 +1172,7 @@ def main():
             )
         else:
             item = results[0]
+            item.unlock()
```

Comment:
Could you add a call to `item.ensure_not_locked()` here (or loop until `unlock()` returns `False`)?

Reply:
Done, added "busy-waiting" for `unlock` to return `False` with a message to confirm the unlock when promted